src/java/com/wolfssl/provider/jsse/WolfSSLImplementSSLSession.java

```diff
@@ -647,7 +647,7 @@ public synchronized Principal getPeerPrincipal()
 
     @Override
     public Principal getLocalPrincipal() {
-
+    /* Logic needs to be added to check for client auth when wrapper is made TODO */
```

Comment:
Please indent comment to match code level inside method (over 4 spaces)

Reply:
Fixed Indentation.